src/node_sea.cc

```diff
@@ -161,6 +205,60 @@ void IsExperimentalSeaWarningNeeded(const FunctionCallbackInfo<Value>& args) {
       sea_resource.flags & SeaFlags::kDisableExperimentalSeaWarning));
 }
 
+void GetCodeCache(const FunctionCallbackInfo<Value>& args) {
+  if (!IsSingleExecutable()) {
+    return;
+  }
+
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
```

Comment:
There's no need to get the isolate via an Environment:

```suggestion
  Isolate* isolate = args.GetIsolate();
```